### DISCUSSION TITLE
如果使用 \includepdf{} 插入承诺书，需要在摘要前加入 \setcounter{page}{1} 纠正页码序号

### DISCUSSION THREAD

**Discussion by dreistein543** [nju-lug/NJUThesis#177] [Show and tell]
之前一直使用草稿模式，没有发现该问题，现在发现模板中摘要页码是从罗马数字 3 开始编号（应该是错误地将封面的页码也计算进去了），由于下周一就是预审，在模板更正前能否提供一个小命令临时解决一下？🙏

**Comment by note286** [comment]
我没复现出你的问题，请你提供相应的复现代码。

```latex
\documentclass{njuthesis}
\begin{document}
\maketitle
\begin{abstract}
\end{abstract}
\begin{abstract*}
\end{abstract*}
\tableofcontents
\mainmatter
\end{document}
```

![image](https://user-images.githubusercontent.com/16207292/219823445-036b467f-5859-4a77-8a0a-f327bca2aad4.png)

**Reply by dreistein543** [reply]
我知道原因在哪了，因为我在前面用 `\includepdf{}` 命令添加了诚信承诺书，应该是把承诺书算进页码中了

```latex
\includepdf[pages={1,{}}]{chapters/承诺书.pdf}
\begin{abstract}
\end{abstract}
```

我在摘要 前面加上 `\setcounter{page}{1}` 可以解决问题

建议在说明书中为这种情况增加说明

**Reply by sastars9** [reply]
你好，现在插入pdf的命令好像没法用了，有新的方式吗？

**Reply by dreistein543** [reply]
需要载入pdfpages宏包，你是不是没有载入？

**Reply by dreistein543** [reply]
<img width="1346" alt="image" src="https://user-images.githubusercontent.com/30745407/228579418-e6d38433-75fd-4078-8902-5ede84f7f07d.png">

**Reply by sastars9** [reply]
谢谢大佬，的确是这样的，不好意思，只看了讨论没查文献

**Reply by atxy-blip** [reply]
> 建议在说明书中为这种情况增加说明

加了，下次有更新内容时一起 push。
